packages/synapse-react-client/src/components/EntityBadgeIcons/EntityBadgeIcons.test.tsx

```diff
@@ -1,43 +1,40 @@
-import { act, render, screen, waitFor } from '@testing-library/react'
```

Comment:
No logical changes to these tests. An unused prop was removed from the component under test. I moved this test file to the `src` folder and fixed type errors.